wled00/FX.cpp

```diff
@@ -7305,9 +7293,12 @@ uint16_t mode_waterfall(void) {                   // Waterfall. By: Andrew Tulin
   if (SEGENV.aux0 != secondHand) {                        // Triggered millis timing.
     SEGENV.aux0 = secondHand;
 
-    //uint8_t pixCol = (log10f((float)FFT_MajorPeak) - 2.26f) * 177;  // 10Khz sampling - log10 frequency range is from 2.26 (182hz) to 3.7 (5012hz). Let's scale accordingly.
-    uint8_t pixCol = (log10f(FFT_MajorPeak) - 2.26f) * 150;           // 22Khz sampling - log10 frequency range is from 2.26 (182hz) to 3.967 (9260hz). Let's scale accordingly.
-    if (FFT_MajorPeak < 182.0f) pixCol = 0;                           // handle underflow
+#if defined(CONFIG_IDF_TARGET_ESP32C3) || defined(CONFIG_IDF_TARGET_ESP32S2)
+    uint8_t pixCol = (log10f(FFT_MajorPeak) - 2.26f) * 177;   // 16Khz sampling - log10 frequency range is from 2.26 (182hz) to 3.85 (7079hz). Let's scale accordingly.
+#else
+    uint8_t pixCol = (log10f(FFT_MajorPeak) - 2.26f) * 150;   // 22Khz sampling - log10 frequency range is from 2.26 (182hz) to 3.967 (9260hz). Let's scale accordingly.
+#endif
+    if (FFT_MajorPeak < 182.0f) pixCol = 0;                   // handle underflow
```

Comment:
what is this about - it seems unrelated to the merge of simulateSound and getAudioData?

Reply:
Confirmed-its unrelated. 
The code tried to adjust this effect's frequency range for -S2 and -C3.  Which is not so bad, if -S2/-C3 use a different sampling rate in audioreactive. 

Reminds me that I want to clean up this log10() chaos in fx.cpp: 1. use logf() instead of log10f() 2. Use constants instead of magic numbers.
But that's a different PR.